specs/signature-specification.md

```diff
@@ -155,8 +155,8 @@ These attributes are considered unsigned with respect to the signing key that ge
 ## OCI Signatures
 
 This section describes how a Notary Project signature is stored in an OCI Distribution conformant registry.
-OCI image manifest is used to store signatures in the registry, see [OCI image spec v1.1.0-rc3][oci-image-manifest] for details.
-The signature manifest has a configuration media type that specifies it's a Notary Project signature, a subject referencing the manifest of the artifact being signed, a layer referencing the signature, and a collection of annotations.
+OCI image manifest is used to store signatures in the registry, see [OCI image spec v1.1.0][oci-image-manifest] for details. Notary Project signature is compliant with OCI image spec v1.1.0.
+Starting from `notation` v2.0.0, the signature manifest uses the `artifactType` property that specifies it's a Notary Project signature, a subject referencing the manifest of the artifact being signed, a layer referencing the signature, and a collection of annotations.
```

Comment:
Can `notation` 2.0.0 opt to produce the old-style manifest with a toggled flag?

Reply:
@yizha1 @shizhMSFT Updated per your suggestions. Thanks.
I will raise another PR to update `notation` CLI to include the new flag.